test/call-super-in-callbacks/call-super-in-callbacks.html

```diff
@@ -80,7 +80,7 @@
    * @mixes MixinConnected
    * @polymerElement
    */
-  class GoodMixin1 extends MixinConnected(HTMLElement) {
+  class GoodMixin2 extends MixinConnected(HTMLElement) {
```

Comment:
why?

Reply:
Babel's scope analysis crashes when there are multiple classes with the same name in the same scope(this is illegal)